### PR TITLE
Initial cut.

### PR DESCRIPTION
**Pull request by timlesallen**

### CODE REVIEW

File: src/get-password-feedback.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Use the globally available zxcvbn lib to calculate a passwords strength.
```
Comment (timlesallen) [inline src/get-password-feedback.js]:
```suggestion
 * Use the globally available zxcvbn lib to calculate a password's strength.
```

File: src/load-script-once.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    scriptCache[src] = new Promise((resolve, reject) => {
+      const callback = err => err ? reject(err) : resolve();
+      loadScript(src, { attrs: { ...(integrity && { integrity }), crossorigin: 'anonymous' } }, callback);
+    });
+  }
+
+  return scriptCache[src];
+}
+
+export default loadScriptOnce;
```
Comment (timlesallen) [inline src/load-script-once.js:23]: I would have thought someone had already done this exact thing on npm?

Comment (timlesallen) [reply src/load-script-once.js:23]: In fact, you've got this on package.json? https://www.npmjs.com/package/load-script-once

Comment (aeh) [reply src/load-script-once.js:23]: @timlesallen I know and I did try using that but it does not have the ability to pass options through... so if I was to use that we could not set the cors anonymous or integrity hash attributes.

Comment (timlesallen) [reply src/load-script-once.js:23]: @aeh Ah!  Maybe publish yours then? :P

**Comment by timlesallen** [comment]
@aeh LGTM!

**Pull request merged**